src/Common/src/System/Collections/Generic/LowLevelDictionary.cs

```diff
@@ -55,7 +55,7 @@ public int Count
 
                 Entry entry = Find(key);
                 if (entry == null)
-                    throw new KeyNotFoundException();
+                    throw new KeyNotFoundException(SR.Format(SR.Arg_KeyNotFoundWithKey, key.ToString())); 
```

Comment:
This should not be added to LowLevelDictionary. The whole point of LowLevelDictionary is that it does not have all the convenience cruft from regular dictionary.